packages/appstore/src/components/onboarding-new/static-get-more-accounts.tsx

```diff
@@ -13,7 +12,7 @@ const StaticGetMoreAccounts = ({ icon, description, title }: IGetMoreAccounts) =
     return (
         <div className='static-get-more-accounts'>
             <div>
-                <Icon icon={icon} size={32} className='static-get-more-accounts__icon' />
+                <Icon data_testid='dt_icon' icon={icon} size={32} className='static-get-more-accounts__icon' />
```

Comment:
Can you just mock the icon component instead of adding data-testid?

Reply:
done